### PR TITLE
Fix gcc compiler error

### PR DESCRIPTION
**Pull request by RussTreadon-NOAA**
The gcc compiler requires a field width specification when formatting numeric output.  Automated gcc testing flagged the line

```
     write(6,'(a,i,a)')  'Command line input for ',iargc(),' arguments'
```
in `src/EnKF/gfs/src/getsigensmeanp_smooth.fd/getsigensmeanp_smooth_ncep.f90`

This PR adds a field width to the integer format so that the above line reads
```
     write(6,'(a,i3,a)')  'Command line input for ',iargc(),' arguments'
```

With this change in place automated gcc testing passes.

### CODE REVIEW

**Comment by RussTreadon-NOAA** [comment]
@CoryMartin-NOAA and @DavidNew-NOAA , automated gcc testing flagged the formatting error described above.  This PR adds a field width to the `i` format.  This allows the gcc test to pass.  I'll merge into `develop` once it is reviewed and approved.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Comment by RussTreadon-NOAA** [comment]
Thank you @CoryMartin-NOAA

**Pull request merged**
